go/tests/test.sh

```diff
@@ -1,21 +1,18 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
```

Comment:
❓ What's the significance of `#/` vs `#`? Genuinely curious.

Reply:
💡 [Usage output](https://github.com/habitat-sh/core-plans/pull/2603/files#diff-c95c957fa636e68f1c72a1219c1d5b4aR9)!  That's clever. 😃